src/lib/flexbox/api/base.ts

```diff
@@ -92,6 +91,15 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
   // Lifecycle Methods
   // *********************************************
 
+  /**
+   * Use post-component-initialization event to perform extra
+   * querying such as computed Display style
+   */
+  ngOnInit() {
+    this._display = this._getDisplayStyle();
+    this._hasInitialized = true;
```

Comment:
`ngOnChanges()` is called 1x before ngOnInit() and then later when the @Input values are externally changed. The intention of that logic is to SKIP any responses to changes until after `ngOnInit()`.